doc/md/versioned-migrations.mdx

```diff
@@ -85,30 +141,75 @@ import (
 func main() {
 	ctx := context.Background()
 	// Create a local migration directory able to understand golang-migrate migration files for replay.
-	dir, err := sqltool.NewGolangMigrateDir("migrations")
+	dir, err := sqltool.NewGolangMigrateDir("ent/migrate/migrations")
 	if err != nil {
 		log.Fatalf("failed creating atlas migration directory: %v", err)
 	}
-	// Write migration diff.
+	// Migrate diff options.
 	opts := []schema.MigrateOption{
 		schema.WithDir(dir),                         // provide migration directory
 		schema.WithMigrationMode(schema.ModeReplay), // provide migration mode
 		schema.WithDialect(dialect.MySQL),           // Ent dialect to use
 	}
-	// Generate migrations using Atlas support for TiDB (note the Ent dialect option passed above).
-	err = migrate.NamedDiff(ctx, "tidb://user:pass@localhost:3306/ent_dev", "my_migration", opts...)
+	if len(os.Args) != 2 {
+		log.Fatalln("migration name is required. Use: 'go run -mod=mod ent/migrate/main.go <name>'")
```

Comment:
should it be required?

Reply:
In this migration example yes. I think it makes sense.